packages/webpack/src/ember-webpack.ts

```diff
@@ -76,6 +66,7 @@ const Webpack: Packager<Options> = class Webpack implements PackagerInstance {
   private extraConfig: Configuration | undefined;
   private passthroughCache: Map<string, Stats> = new Map();
   private publicAssetURL: string | undefined;
+  private extraThreadLoaderOptions: object | false | undefined;
```

Comment:
maybe you can use ThreadLoaderOptions Type instead of object

Reply:
I couldn't find types for `thread-loader` anywhere, and wasn't sure we'd want to sign up for maintaining our own type. We certainly could though...

Reply:
Oh, I couldn't find either, nvm..